dockerfiles/noetic/base/dockerfile

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+FROM ros:noetic-perception-focal
```

Comment:
This was because the perception contains opencv so we don't need to build it ourselves in noetic. In kinetic we are using python3 so the installed version of opencv won't work. I am mostly convinced we need to build ROS from scratch here if we are using python3 so starting from a ros dockerfile is likely not going to work anyway.